Source/ProgressionSystemRuntime/Private/Widgets/PSEndGameWidget.cpp

```diff
@@ -33,6 +35,9 @@ void UPSEndGameWidget::NativeConstruct()
 
 	// Binds the local player state ready event to the handler
 	BIND_ON_LOCAL_PLAYER_STATE_READY(this, ThisClass::OnLocalPlayerStateReady);
+
+	UPSWorldSubsystem& WorldSubsystem = UPSWorldSubsystem::Get();
+	WorldSubsystem.OnCurrentScoreChanged.AddUniqueDynamic(this, &ThisClass::UpdateProgressionWidgetForPlayer);
```

Comment:
1. Please rename `UpdateProgressionWidgetForPlayer()` to `OnCurrentScoreChanged()`, it's very frustrating when delegated function called differently. On the other hand, when callback functionis called the same as delegate, others immidiately understand that its called by delegate
2. Also, make this function BlueprintNativeEvent accessible for blueprint-developers, since it's event-driven function called from delegate
3. Repeat 1 and 2 the same for another widget where you bind: PSOverlayWidget.cpp line 83

Reply:
done: https://github.com/h4rdmol/ProgressionSystem/pull/45/commits/7b22bd4cb4992c876abf63ded7e9108d55c074ea
https://github.com/h4rdmol/ProgressionSystem/pull/45/commits/04bd55edff61e3040611078f0f6de4dc86185189